src/components/organisms/top/top.html

```diff
@@ -0,0 +1,20 @@
+<div class="top">
+  <div class="img">
```

Comment:
```suggestion
  <div class="img-wrapper">
```
この要素は `img` 要素自体ではないので、クラス名でそのように明示してあげると可読性が上がります。